desktop-src/dataxchg/html-clipboard-format.md

```diff
@@ -21,244 +21,346 @@ ms.date: 05/31/2018
+### The fragment, the selection, and their context
 
-The selection is optional, as sufficient information is included in the fragment for basic pasting. If the selection is not stored, both StartSelection and EndSelection are not stored in the header.
-
-The context is a valid, complete HTML document. This article contains the fragment and all preceding surrounding tags (start and end tags; these preceding surrounding tags represent all the parent nodes of the fragment, until the HTML node). The article also contains the complete HEAD, and allows the BASE and TITLE elements, for example, to be included so this additional information can be obtained. An application copying a fragment of HTML to the clipboard can choose to create a BASE element to include in the context if such an element is not already present so that partial URLs in the fragment can be resolved.
-
-The context is optional, as sufficient information is included in the fragment for basic pasting of an HTML fragment to take place. If the context is not stored, the fragment only is stored and the StartHTML=EndHTML=-1.
-
-## Scenarios
-
-The following scenarios describe how the IE4/MSHTML HTML editor handles HTML cut and paste; other applications may or may not follow these scenarios. The clipboard format described here is intended to allow flexibility for how an application chooses to function. (These scenarios show only good HTML, that is, no overlapping tags.)
+| Element   | Description headers                 | Must be well-formed HTML? | Start and End character positions must
+|-----------|-------------------------------------|---------------------------|
+| Context   | `StartHTML` and `EndHTML`           | Yes						  |
+| Fragment  | `StartFragment` and `EndFragment`   | Yes						  |
+| Selection | `StartSelection` and `EndSelection` | No						  |
 
-1.  Simple Fragment of HTML.
-    -   -   HTML text:
+#### Context
 
-            &lt;BODY&gt;This is normal <B>This is bold </B><I><B>This is bold italic </B>This is italic </I>&lt;/BODY&gt;
+The _context_ is a valid, complete HTML document - though this does not mean the entire original source HTML document containing the user's selection will be carried-over verbatim; on the contrary, it can be a minimal, but well-formed, HTML document).
 
-        -   Appears as:
+This _context_ contains the _fragment_ and all preceding surrounding tags (start and end tags; these preceding surrounding tags represent all the parent nodes of the fragment, until the HTML node). Observe that the above example article has a complete HTML `<head>` element which permits the use of `<base href="">` and `<title>` elements, for example, to be included so this additional information can be obtained. An application copying a fragment of HTML to the clipboard can choose to create a `<base href="">` element to include in the context if such an element is not already present so that non-absolute URIs in the HTML fragment can be resolved.
```

Comment:
```suggestion
This _context_ contains the _fragment_ and all preceding surrounding tags (start and end tags; these preceding surrounding tags represent all the parent nodes of the fragment, until the HTML node). The above example article has a complete HTML `<head>` element which permits the use of `<base href="">` and `<title>` elements. For example, this element can be inserted to obtain this additional information. An application copying a fragment of HTML to the clipboard can choose to create a `<base href="">` element to include it in the context if such an element is not already present. This way, non-absolute URIs in the HTML fragment can be resolved.
```

This was hard to read, and I think there was an "it" missing - splitting into multiple sentences for readability. Please check if I still match the original idea of your text.

Reply:
I think this is easier to read, but I'll wait for @daiplusplus to chime in before committing the suggestion.